src/okareo/model_under_test.py

```diff
@@ -368,6 +372,19 @@ def _get_test_run_payload(
             checks=checks if checks else UNSET,
         )
 
+    def _handle_scenario_data_point(
```

Comment:
Why do we need this? Can there be a better name?

Reply:
This method just unpacks the `input` field from the scenario data point, which can either be a `ScenarioDataPoinResponseType0` or a Python type. It is logic that is needed in both batch/non-batch paths, so I moved it into this helper method.

I changed the name to `_extract_input_from_scenario_data_point` and added a small docstring. Lmk if that looks okay.